README.md

```diff
@@ -153,6 +153,12 @@ You can run tests to check for linter errors by running the following commands:
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
thanks for the highlights, i will make those changes right away

Reply:
right now I dont have a logo, but I will consider that. Thank you

Reply:
Sure, it's optional but it is highly recommended dear 👍